src/farms/AvailableFarms.ts

```diff
@@ -28,6 +28,9 @@ export const INK_GRANS_FarmContract = "0xC4f628150EaDcA9864641e3BF65F8Ea4Fd75e23
 export const INK_ESCH_LPAddress = "0x4030c26348628e1eeb11955c68e651c7027b04f9";
 export const INK_ESCH_FarmContract = "0x6E59E5cd333CE71D3AFDEdae09949729dC2fe4B3";
 
+export const UBQ_ESCH_LPAddress = "0xf102ad140b26c3c6af9e9358da9deaa27cb1dbea";
```

Comment:
we have this address already in another variable in this file

Reply:
duh :)

thanks